e2e/scheduler_test.go

```diff
@@ -73,7 +74,8 @@ func testScheduler() {
 
 		ns := testNamespacePrefix + randomString(10)
 		createNamespace(ns)
-		stdout, stderr, err := kubectlWithInput([]byte(fmt.Sprintf(capacityPodTemplateYAML, ns, "1073741824")), "apply", "-f", "-")
+		stdout, stderr, err := kubectlWithInput([]byte(fmt.Sprintf(capacityPodTemplateYAML, ns, topolvm.GetCapacityKeyPrefix(), "1073741824", topolvm.GetPluginName(), topolvm.GetPluginName())),
```

Comment:
```suggestion
		stdout, stderr, err := kubectlWithInput([]byte(fmt.Sprintf(capacityPodTemplateYAML, ns, topolvm.GetCapacityKeyPrefix(), "1073741824", topolvm.GetCapacityResource(), topolvm.GetCapacityResource())),
```

In addition, the corresponding format specifier should be replaced from `%s/capacity` to `%s`.

Reply:
nits: this line seems too long to read. It may be nice to split.

Reply:
Same as https://github.com/topolvm/topolvm/pull/665#discussion_r1123908177

> nits: this line seems too long to read. It may be nice to split.

Thanks. I will split it afther https://github.com/topolvm/topolvm/pull/665#discussion_r1123908177 is resolved.

Reply:
Fixed.